deploy/ansible/deployandtest.sh

```diff
@@ -52,6 +52,12 @@ then
     exit 1
   fi
   VARIABLES="$VARIABLES project=$GCP_PROJECT"
+
+  if [ -n "$GCP_SSH_KEY" ]
```

Comment:
Probably good to name it `GCP_SSH_KEY_FILENAME` to be clear?

Reply:
Done.